modules/planning/tasks/deciders/path_bounds_decider/path_bounds_decider.cc

```diff
@@ -213,7 +213,19 @@ Status PathBoundsDecider::Process(
                << std::get<3>(pull_over_configuration) << "] theta["
                << std::get<4>(pull_over_configuration) << "]";
         // Trim the path bound based on the pull-over s.
-        // TODO(jiacheng): implement this.
+        int pull_over_pos_idx =
+            std::max(std::get<5>(pull_over_configuration),
+            kNumExtraTailBoundPoint);
+        for (auto& path_boundary : candidate_path_boundaries) {
```

Comment:
@panjiacheng Would it be more efficient to use `vector::erase` and directly operate on the `candidate_path_boundaries` ?